upstash/redis/database/crud.go

```diff
@@ -123,14 +121,16 @@ func resourceDatabaseCreate(ctx context.Context, data *schema.ResourceData, m in
 		}
 	}
 
+	region := data.Get("region").(string)
+	if region != "global" && region != "us-central1" {
+		return diag.Errorf("Regional databases are deprecated. The only region configuration is `global`.")
```

Comment:
... configuration is global or gcp us-central1

Reply:
Actually, I am not sure if we should handle this explicitly in the provider. We already return an err on the API side.

Reply:
We would also have to change all references of regional dbs
```
Description: "region of the database. Possible values are: " +
```
is one example